_includes/install/flow-diagram.md

```diff
@@ -16,7 +16,7 @@ The diagram shows the following:
 
 	*	For simplicity, get a compressed {{site.data.var.ce}} or {{site.data.var.ee}} [archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html), extract it on your Magento server, and start your installation.
 
-	*	If you are more technical and you are familiar with Composer, get a {{site.data.var.ce}} or {{site.data.var.ee}} [metapackage]({{ page.baseurl }}/install-gde/prereq/integrator_install.html).
+	*	If you are more technical and you are familiar with Composer, get a {{site.data.var.ce}} or {{site.data.var.ee}} [metapackage]({{ page.baseurl }}/install-gde/composer.html).

```

Comment:
Possible 2.0 using include? If used will 404 when clicked in 2.0 pages.

Reply:
Yes, a search for references to these includes in the 2.0 files did return results. I'll make changes.